community/sample-apps/react-relay/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
```

Comment:
This feels like more than a sample app :)

Reply:
This stuff is auto-generated by create-react-app